pkg/wasm/httpfetcher.go

```diff
@@ -23,39 +25,53 @@ import (
 	"github.com/cenkalti/backoff/v4"
 )
 
+var (
+	httpInitialBackoff = time.Millisecond * 500
+	defaultHTTPFetcher = NewHTTPFetcher()
```

Comment:
Why not make it a field of the cache?  package var is notencouraged

https://dave.cheney.net/2017/06/11/go-without-package-scoped-variables

Reply:
As I mentioned before, `httpInitialBackoff` is basically just a constant. But when testing, it is need to be shorten for making easy the test. Therefore, it is defined as a package variable.
In case of `defaultHTTPFetcher` is a singleton. 

Do we need to include them for each instance of caches? 
WDYT?

Reply:
I am not sure why each instance of caches, there is only one cache instance

Reply:
Yes. it is true. I just thought about loosening the coupling between cache and fetcher.
BTW, since I agree with that the change is not necessary and may be not-good in some sense, so I moved it back to the field of the cache.

Thanks!